dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -342,6 +345,8 @@ class ScriptSeedTest < ActiveSupport::TestCase
 
     test 'seed updates lesson programming expressions' do
       script = create_script_tree
+      CourseOffering.add_course_offering(script)
```

Comment:
Why does this test need course offering?

Reply:
This was an unintentional result of a bad merge. I'm not really sure whether it is best to include it or remove it. It isn't necessary for the test to pass, so I've removed it. Probably this means the script being tested will have no resources or vocab, so we aren't testing that resources and vocab are preserved, but that seems a bit weirdly specific to worry about testing here.